config/isis_framewave.py

```diff
@@ -0,0 +1,109 @@
+import click
+import utilities_common.cli as clicommon
+
+from swsssdk import ConfigDBConnector
```

Comment:
>swsssdk [](http://example.com/codeflow?start=5&length=7)

swsssdk is deprecated by swsscommon. Please use it with new code on master branch.